docs/contributors/contributors-guide.md

```diff
@@ -195,6 +195,4 @@ To learn more about these features, see [Understanding Lists Data](../maintainer
 
 ## Conclusion
 
-OpenSauced is a platform dedicated to empowering you, the contributor, in your open source journey. With OpenSauced, you can find repositories to contribute to, track your contributions, gain insights into your favorite open source projects, and find new contributors to connect with.
```

Comment:
@BekahHW I removed this paragraph as it's duplicate with the intro paragraph. Feel free to move it back in if that's what you've intended. 🙂